package.json

```diff
@@ -1,21 +1,21 @@
 {
   "dependencies": {
-    "@azure-tools/typespec-autorest": "^0.48.0",
-    "@azure-tools/typespec-azure-core": "^0.48.0",
-    "@azure-tools/typespec-azure-resource-manager": "^0.48.0",
-    "@azure-tools/typespec-client-generator-core": "^0.48.4",
-    "@typespec/compiler": "^0.62.0",
-    "@typespec/http": "^0.62.0",
-    "@typespec/http-client-csharp": "^0.1.9-alpha.20241211.3",
-    "@typespec/http-server-csharp": "^0.58.0-alpha.5",
-    "@typespec/http-server-javascript": "file:./pkg/typespec-http-server-javascript-0.58.0-alpha.5.tgz",
```

Comment:
Could you remove folder directly https://github.com/allenjzhang/typespec-e2e-demo/tree/main/pkg?